projects/hslayers/src/components/add-data/file/raster-images/raster-images.component.ts

```diff
@@ -25,7 +26,8 @@ export class HsFileGeotiffComponent
   }
   ngOnInit(): void {
     this.baseDataType = this.dataType;
-    this.acceptedFormats = '.tif, .tiff, .gtiff, .zip';
+    this.acceptedFormats =
+      '.tif, .tiff, .tfw, .png, .pgw, .png.aux.xml, .jpeg, .jgw, .jpg.aux.xml, .jp2, .j2w, .zip';
```

Comment:
As per https://en.wikipedia.org/wiki/World_file#Filename_extension I would extend this to
`'.tif, .tifw, .tiff, .tiffw, .tfw, .png, .pngw, .pgw, .png.aux.xml, .jpg, .jpgw, .jpeg, .jpegw, .jgw, .jpg.aux.xml, .jp2, .jp2w, .j2w, .zip', .wld`

Also the common `.jpg` was missing.

Reply:
@jmacura Will include these changes, just wasn't sure if Layman supports them all, as they were wont mentioned in the [rest.md file](https://github.com/LayerManager/layman/blob/master/doc/rest.md#post-workspace-layers)

Reply:
@DailisLangovskis Oh, well, yes, then this shall be a feature request for the Layman team. @fzadrazil ?
Perhaps @jirik could give us some hint whether the list of extensions in their [REST documentation](https://github.com/LayerManager/layman/blob/master/doc/rest.md#post-workspace-layers) is exhaustive.

It would be of course unpleasant if we accepted something which would be refused on the Layman's side...

Reply:
.jpg was left out by accident

Reply:
> As per https://en.wikipedia.org/wiki/World_file#Filename_extension I would extend this to `'.tif, .tifw, .tiff, .tiffw, .tfw, .png, .pngw, .pgw, .png.aux.xml, .jpg, .jpgw, .jpeg, .jpegw, .jgw, .jpg.aux.xml, .jp2, .jp2w, .j2w, .zip', .wld`
> 
> Also the common `.jpg` was missing.

I think the list is accurate like this, but I honestly haven't tried all of them for the lack of sample data. I will confirm this with @jirik on Tuesday.

Reply:
We tested only files mentioned in REST API documentation. Additional files .tifw, .tiffw, .pngw, .jpgw, .jp2w, .wld were not tested, but there is quite a chance Layman will accept it and gdal will detect georeferencing from them. I guess file .jpeg will not be recognized by Layman as main file currently.

Reply:
Updated the accesptedFormats predefined array

Reply:
@DailisLangovskis @jirik @DailisLangovskis Tested now with `.pngw` and `.wld`. Both went fine 👍